metadata-ingestion/src/datahub/ingestion/source/looker/looker_config.py

```diff
@@ -298,6 +298,17 @@ class LookerDashboardSourceConfig(
         description="When enabled, platform instance will be added in dashboard and chart urn.",
     )
 
+    folder_path_pattern: AllowDenyPattern = Field(
+        default=AllowDenyPattern.allow_all(),
+        description="Filter for specific looker folder. "
+        "For example: \n"
```

Comment:
This comment is super long. I'll let Harshal decide what to do here

Reply:
seems like it's missing a space in the middle, but doesn't look too long

can probably just say "Dashboards will only be ingested if they're allowed by both this config and `dashboard_pattern`"

<img width="873" alt="image" src="https://github.com/user-attachments/assets/c2d6cdcb-40b1-4824-861f-7e58078fcdaa">

Reply:
done